src/main/java/com/google/step/datamanager/RestaurantPlaceManager.java

```diff
@@ -0,0 +1,10 @@
+package com.google.step.datamanager;
+
+import java.util.List;
+import java.util.Set;
+
+public interface RestaurantPlaceManager {
+  public void updatePlacesOfRestaurants(long restaurantId, List<String> placeIds);
```

Comment:
```suggestion
  public void updatePlacesOfRestaurant(long restaurantId, List<String> placeIds);
```

There's only a single restaurantId?